pyscriptjs/src/plugin.ts

```diff
@@ -74,11 +74,23 @@ export class PluginManager {
     }
 
     beforeLaunch(config: AppConfig) {
-        for (const p of this._plugins) p.beforeLaunch(config);
+        for (const p of this._plugins) {
+            try{
+                p.beforeLaunch(config);
+            } catch(e) {
+                logger.error('Error while calling beforeLaunch hook', e);
```

Comment:
Possible to add a reference to the specific plugin here? "Error while calling beforeLaunch hook of plugin 'p'" or something?